pages/demos/snackbars.js

```diff
@@ -3,7 +3,7 @@ import 'docs/src/modules/components/bootstrap';
 import React from 'react';
 import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
 
-const req = require.context('docs/src/pages/demos/snackbars', false, /\.md|\.js$/);
+const req = require.context('docs/src/pages/demos/snackbars', false, /\.(md|js|tsx)$/);
```

Comment:
I guess this has to be applied to every demo?

Reply:
Yes, I was waiting on a preliminary approval.